parquet-hadoop/src/test/java/org/apache/parquet/filter2/recordlevel/TestRecordLevelFilters.java

```diff
@@ -255,6 +255,7 @@ public void testComplex() throws Exception {
     DoubleColumn lat = doubleColumn("location.lat");
 
     FilterPredicate pred = or(and(gt(lon, 150.0), notEq(lat, null)), eq(name, Binary.fromString("alice")));
+//    FilterPredicate pred = eq(name, Binary.fromString("alice"));
```

Comment:
I has simplified this test to make debugging easier, I have removed this line as it is just a sub-expression in the larger one and is covered by the original test.